### PR TITLE
Tsport tsdoc

### PR DESCRIPTION
**Pull request by RyanGuild**
Added the typescript files to the autodocumentation

### CODE REVIEW

File: 3Dmol/Gradient.ts [hunk 1]
```diff
@@ -207,21 +222,21 @@ export class Sinebow extends GradientType {
...
+    this.min = min;
+    this.max = max;
+    if (typeof max == "undefined" && Array.isArray(min) && min.length >= 2) {
+      //we were passed a single range
+      this.max = min[1];
+      this.min = min[0];
+    }
+    if (max < min) {
+      //reverse the order
+      this.mult = -1.0;
```
Comment (RyanGuild) [inline 3Dmol/Gradient.ts:235]: nothing changed here

File: 3Dmol/colors.ts [hunk 2]
```diff
@@ -155,49 +155,34 @@ export const htmlColors = {
...
-                  viewer.setBackgroundColor(0xffffffff);
-                  var colorAsSnake = function(atom) {
-                    return atom.resi % 2 ? 'white': 'green'
-                  };
-
-                  viewer.setStyle( {chain:'A'}, { cartoon: {colorfunc: colorAsSnake }});
-                  viewer.setStyle( {chain:'B'}, { stick: {colorscheme: 'yellowCarbon'}});
-
-                  viewer.render();
-              });
```
Comment (RyanGuild) [inline 3Dmol/colors.ts:181]: this same test is repeated elsewhere in the file

File: package.json [hunk 3]
```diff
@@ -21,6 +21,7 @@
     "@babel/plugin-transform-typescript": "^7.18.8",
     "@babel/polyfill": "^7.12.1",
     "@babel/preset-env": "^7.18.6",
+    "@types/estree": "^1.0.0",
```
Comment (RyanGuild) [inline package.json:24]: required for `npx typedoc`

**Review by RyanGuild — COMMENTED** [review]
looks ready to go

**Review by dkoes — APPROVED** [review]

**Pull request merged**